IM_calculation/Advanced_IM/advanced_IM_factory.py

```diff
@@ -54,6 +54,9 @@ def compute_ims(accelerations, configuration, adv_im_out_dir):
             ]
             # waveform component sequence
             comp_list = ["000", "090", "ver"]
+            if "required_components" in im_config:
+                comp_list = im_config["required_components"]
```

Comment:
Could be 
comp_list = im_config.get("required_components", ["000", "090", "ver"])

Reply:
I can't quite remember either - I thought optional ver component could work, but looking at the code again, I can't think of an elegant solution for that. Perhaps requirement_components can be taken from advanced_im_config.yaml into runlibs_2d.py as we do in advanced_IM_factory.py and we could make parse_args() more intelligent. 
Other than that, I really don't see why we should enable ver component for SDI

Reply:
In fact, we don't need --OpenSees_path for SDI either, but  it is added regardless. It does no harm, but pointless nonetheless